trimsock.gd/addons/trimsock.gd/reactors/tls_client_reactor.gd

```diff
@@ -0,0 +1,36 @@
+extends TrimsockReactor
+class_name TrimsockTLSClientReactor
```

Comment:
Refactor this, if possible, so it's not an almost 1:1 copy of TrimsockTCPClientReactor